src/index.html

```diff
@@ -1,12 +1,319 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img class="logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="is-active nav__link"> Apple </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#"
+              class="nav__link"
+              data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="card" data-qa="card">
+        <a href="#" class="card__img">
+          <img
+            src="images/imac.jpeg"
+            alt="imac"
+            class="card__imac"
+          >
+        </a>
+        <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__rating">
+          <div class="rating card__rating--stars">
+            <div class="rating card__rating--element"></div>
+            <div class="rating card__rating--element"></div>
+            <div class="rating card__rating--element"></div>
+            <div class="rating card__rating--element"></div>
+            <div class="rating card__rating--element"></div>
+          </div>
+          <div class="rating card__rating--reviews">Reviews: 5</div>
+        </div>
+        <div class="card__sell">
+          <div class="sell sell--price">Price:</div>
+          <div class="sell sell--cost">$2,199</div>
+        </div>
+        <div class="card__button-buy">
+          <a
+            href="#"
+            class="button-buy"
+            data-qa="card-hover">
+              Buy
+          </a>
+        </div>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```